examples2d/nphysics_testbed2d/src/testbed.rs

```diff
@@ -94,7 +94,7 @@ impl<'a> Testbed<'a> {
             minor_version:      1
         };
         let window =
-            match RenderWindow::new(mode, "nphysics 2d demo", Close, &setting) {
+            match RenderWindow::new(mode, "nphysics 2d demo", Close, &ContextSettings::default()) {
```

Comment:
It's probably best to keep your modification. I believe the `setting` variable initialization just above could be removed now.